realm/realm-library/src/main/java/io/realm/FieldAttribute.java

```diff
@@ -41,7 +41,7 @@
      * null} depends on the type of the field.
      *
      * @see io.realm.annotations.Required
-     * @see RealmObjectSchema#setRequired(String, boolean)
+     * @see StandardRealmObjectSchema#setRequired(String, boolean)
```

Comment:
This should still reference `RealmObjectSchema`. `StandardRealmObjectSchema` is an implementation detail that shouldn't be known to users.